focus-android/quality/detekt.yml

```diff
@@ -596,15 +687,15 @@ style:
   UnusedPrivateMember:
     active: true
     allowedNames: '(_|ignored|expected|serialVersionUID)'
-    ignoreAnnotated: ['Composable']
+    ignoreAnnotated: ['Preview'] # Configuration for Compose https://github.com/mozilla-mobile/android-components/issues/11866
   UseAnyOrNoneInsteadOfFind:
-    active: false
+    active: true
   UseArrayLiteralsInAnnotations:
-    active: false
+    active: true
   UseCheckNotNull:
     active: true
   UseCheckOrError:
-    active: false
+    active: true
```

Comment:
Why make this true? I see that [the detekt team](https://github.com/detekt/detekt/issues/4643) are enabling them default true as a way to promote, but I don't see the value it brings. Do you have a stronger opinion to wanting this updated?

Reply:
See https://github.com/mozilla-mobile/firefox-android/pull/292#discussion_r1054926875